src/doc/python_scripting/functions.rst

```diff
@@ -12175,15 +12188,43 @@ WriteConfigFile
 WriteScript
 -----------
 
+**Synopsis:**
+
+::
+
+  WriteScript(f)
+
+f : file
+    The python file object that will be written to.
+
+**Description:**
+
+    WriteScript is meant to be able to write out the current state of VisIt to a Python script that can be used later to reproduce a visualization. 
+    This is like using a session file but you can take the output of WriteScript and further customize it.
+
 
 **Example:**
 
 ::
 
-  f = open('script.py', 'wt')
+  #% visit -cli
+  f = open("script.py", "wt")
   WriteScript(f)
   f.close()
 
+The resulting script will contain commands to set up plots in any visualization window that contained plots when WriteScript was called. 
+You may want to edit the resulting script file since it may be verbose, depending on how many plots you have or the complexity of the data file. For example, you might want to remove the code related to setting a plot's SIL restriction.
```

Comment:
```suggestion
The resulting script may be more verbose than necessary. Users may find it useful to delete portions of the script that are not needed.  This will depend on how many plots there are or the complexity of the data. For example, it might useful to remove code related to setting a plot's SIL restriction.
```